seeds/raceSeeds.js

```diff
@@ -5,31 +5,35 @@ const raceData = [
     title: "Lee USA Speedway",
     location_town: "Lee",
     location_state: "NH",
-    owner_id: "1"
+    owner_id: "1",
+    track_image: "https://image.freepik.com/free-vector/suzuka-circuit-motorsport-autosport-japan-grand-prix-race-track_97886-5398.jpg"
   },
   {
     title: "New Hampshire Motor Speedway",
     location_town: "Loudon",
     location_state: "NH",
-    owner_id: "2"
+    owner_id: "2",
+    track_image: "https://image.freepik.com/free-vector/istanbul-circuit-motorsport-autosport-turkey-grand-prix-race-track_97886-5379.jpg"
   },
   {
     title: "New England Dragway",
     location_town: "Epping",
     location_state: "NH",
-    owner_id: "3"
```

Comment:
This should not be deleted - though it won't impact the final product deployed - let's keep this